service/header/service.go

```diff
@@ -44,3 +44,10 @@ func (s *Service) Stop(context.Context) error {
 	log.Info("stopping header service")
 	return nil
 }
+
+// TODO(@Bidon15): Issue #351
```

Comment:
Does this belong to the `IsSyncing` method?

Reply:
We have decided to make it a dedicated subscription API that is related to sync but not to IsSyncing itself